core/mempool/src/lib.rs

```diff
@@ -26,15 +29,16 @@ pub struct Pool {
     // transactions need to be grouped by account, and within each account,
     // ordered by nonce so that runtime will not ignore some of the transactions
     transactions: RwLock<HashMap<AccountId, BTreeMap<u64, SignedTransaction>>>,
+    // store the map that allows us to lookup transaction by hash
+    transaction_info: RwLock<HashMap<CryptoHash, (AccountId, u64)>>,
```

Comment:
Please expand comment to explain exactly what it maps to what: transaction hash -> (account id of the sender, nonce).

Reply:
Maybe create `type Nonce = u64`.